### DISCUSSION TITLE
Need help for putting bolt together

### DISCUSSION THREAD

**Discussion by maichongju** [bitcraze/discussions#168] [Q&A]
I am planning to build the bolt with some third-party components. I solder everything together and test it. But somehow it fired the power controller chip on the Bolt. I just can't figure out what went wrong. 

Components list:
- Battery: Tattu R-Line 550mAh 7.4V 95C 2S
- ESC: Flash hobby 7A 1-2S ESC
- Motor: RCINPOWER GTS V2 1202.5 6000KV MOTOR

So I solder everything together at first and then plug the connection into the bolt. Then I plug in the battery, hear the instance crack from the colt and smell something burning. And then, I can see a little hole on the power chip for the connector for the bolt. 

Then to ensure it is not the motor causing the problem. We switch to a new ESC and only solder the ESC with the connection to the bolt. Then plug the battery in, and the same thing happens again. This time I hear the crack and can smell the burn. But no hole is visible this time. 

Note: We only plug in a single connection into the bolt at all times, and the other three motor connections is unplugged. But I don't think that will be a problem.

Here is what it looks like for the soldering. 
![20220731_131235](https://user-images.githubusercontent.com/31859452/182038214-b1373581-bac1-4eb4-a6e1-c89f4eba9b16.jpg)

**Comment by knmcguire** [comment]
Hi!

That does not sound good... unfortunately internally at Bitcraze we can't help you much since everybody that works closely with the Bolt is now on holiday

Would it be possible to wait for 2 more weeks, so that I'll ping them once they air back that you need some assistance?

**Reply by maichongju** [reply]
2 Weeks is ok. I will just do more research at the same time. But please keep me posted. 

Thank you for the help

**Reply by tobbeanton** [reply]
I've checked the cables and just found out that the colors of the cables has been switched :no_good:

We will have to replace the Bolts you have. Please send us an email to support@bitcraze.io. If you have any working Bolt 1.1 make sure to switch the red and black cable in the connector.

Sorry!
